TBAnnotationClustering-Swift/ViewController.swift

```diff
@@ -43,7 +43,7 @@ class ViewController: UIViewController, MKMapViewDelegate {
         let toRemove = NSMutableSet(set: before)
         toRemove.minusSet(after as Set<NSObject>)
         
-        dispatch_async(dispatch_get_main_queue()) { () -> Void in
+        NSOperationQueue().addOperationWithBlock() {
```

Comment:
I've seen several occasions where views where modified on a background thread and it did not lead to a crash. It usually results in weird behavior.